scripts/pkg/build_templates/opensearch/opensearch.rpm.spec

```diff
@@ -93,7 +93,7 @@ if ! grep -q '## OpenSearch Performance Analyzer' %{config_dir}/jvm.options; the
    echo '## OpenSearch Performance Analyzer' >> %{config_dir}/jvm.options
    echo "-Dclk.tck=$CLK_TCK" >> %{config_dir}/jvm.options
    echo "-Djdk.attach.allowAttachSelf=true" >> %{config_dir}/jvm.options
-   echo "-Djava.security.policy=file:///usr/share/opensearch/plugins/opensearch-performance-analyzer/config/opensearch_security.policy" >> %{config_dir}/jvm.options
+   echo "-Djava.security.policy=file:///usr/share/opensearch/config/opensearch-performance-analyzer/opensearch_security.policy" >> %{config_dir}/jvm.options
```

Comment:
RPM config folder is at `/etc/opensearch` not `/usr/share/opensearch/config`.

Reply:
Modified to ```/etc/opensearch/```.